python/python/tests/test_transaction.py

```diff
@@ -209,6 +209,8 @@ async def transaction_test(
     args.append({"foo", "bar"})
     transaction.smove(key7, key7, "non_existing_member")
     args.append(False)
+    transaction.sinter([key7, key7])
+    args.append(set())
```

Comment:
is key7 empty at this point?  if you move this up to be after sunionstore (after line 206) you should be able to do sinter with a return value.

Reply:
Will move right after line 206.